tensorflow_addons/seq2seq/README.md

```diff
@@ -60,26 +71,38 @@ import tensorflow_addons as tfa
 
 # TF 2.0, new style
 
-# Build RNN
-#   encoder_outputs: [max_time, batch_size, num_units]
-#   encoder_state: [batch_size, num_units]
-encoder = tf.keras.layers.LSTM(num_units, return_state=True)
-encoder_outputs, state_h, state_c = encoder(encoder_emb_inp)
-encoder_state = [state_h, state_c]
+# Encoder
+encoder = tf.keras.layers.LSTM(num_units, return_sequences=True, return_state=True)
+encoder_outputs, state_h, state_c = encoder(
+    encoder_inputs, mask=tf.sequence_mask(encoder_lengths), training=True
+)
+encoder_state = (state_h, state_c)
+
+# Decoder RNN cell with attention
+attention_mechanism = tfa.seq2seq.LuongAttention(num_units, encoder_outputs)
+decoder_cell = tf.keras.layers.LSTMCell(num_units)
+decoder_cell = tfa.seq2seq.AttentionWrapper(
+    decoder_cell,
+    attention_mechanism,
+    attention_layer_size=num_units,
+    initial_cell_state=encoder_state,
+)
 
 # Sampler
 sampler = tfa.seq2seq.sampler.TrainingSampler()
 
 # Decoder
-decoder_cell = tf.keras.layers.LSTMCell(num_units)
 projection_layer = tf.keras.layers.Dense(num_outputs)
-decoder = tfa.seq2seq.BasicDecoder(
-    decoder_cell, sampler, output_layer=projection_layer)
+decoder = tfa.seq2seq.BasicDecoder(decoder_cell, sampler, output_layer=projection_layer)
```

Comment:
Seems to be a long line, can we wrap it?

Reply:
I actually reformatted the example using `black` to be consistent with the codebase itself.